### PR TITLE
full tasks solution

### PR DESCRIPTION
**Pull request by mkazarian-1**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -137,6 +147,14 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+<!--            <plugin>-->
```
Comment (Elena-Bruyako) [inline pom.xml]: remove comments

File: src/main/java/org/example/bookshop/controller/BookController.java [hunk 2]
```diff
@@ -32,4 +37,17 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto bookRequestDto) {
         return bookService.save(bookRequestDto);
     }
+
+    @PutMapping("/{id}")
+    public BookDto updateBook(@PathVariable Long id,
+                              @RequestBody UpdateBookRequestDto bookRequestDto) {
+        bookRequestDto.setId(id);
```
Comment (Elena-Bruyako) [inline src/main/java/org/example/bookshop/controller/BookController.java]:
```suggestion
```

File: src/main/java/org/example/bookshop/dto/UpdateBookRequestDto.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package org.example.bookshop.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UpdateBookRequestDto {
+    private Long id;
```
Comment (Elena-Bruyako) [inline src/main/java/org/example/bookshop/dto/UpdateBookRequestDto.java]:
```suggestion
```

File: src/main/java/org/example/bookshop/service/impl/BookServiceImpl.java [hunk 4]
```diff
@@ -37,4 +38,14 @@ public BookDto getById(Long id) {
                         .orElseThrow(() ->
                                 new EntityNotFoundException("Can't find book by id:" + id)));
     }
+
+    @Override
+    public BookDto update(UpdateBookRequestDto bookRequestDto) {
+        return bookMapper.toDto(bookRepository.save(bookMapper.toModel(bookRequestDto)));
```
Comment (Elena-Bruyako) [inline src/main/java/org/example/bookshop/service/impl/BookServiceImpl.java]:
```suggestion
        Book book = bookRepository.findById(id)
                .orElseThrow(()...
        bookMapper.updateBookFromDto(bookDto, book);
        return bookMapper.toDto(....
```

File: src/main/resources/db/changelog/changes/01-create-employees-table.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: your_name
```
Comment (Elena-Bruyako) [inline src/main/resources/db/changelog/changes/01-create-employees-table.yaml:4]: nice author)

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comments

File: src/main/resources/db/changelog/changes/01-create-employees-table.yaml (hunk 5 above)
Comment (mkazarian-1) [reply src/main/resources/db/changelog/changes/01-create-employees-table.yaml:4]: Tell me please if it is normal that I can't use boolean here?

              - column:
                  name: is_deleted
                  type: **TINYINT(1)**
                  defaultValueBoolean: false
                  constraints:
                    nullable: false

**Review by MentorMateAcademy — APPROVED** [review]

**Pull request merged**